networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="configuring-ingress-cluster-traffic-lb-allowed-source-ranges"]
+= Configuring ingress cluster traffic using Load Balancer allowed source ranges
+include::_attributes/common-attributes.adoc[]
+:context: configuring-ingress-cluster-traffic-lb-allowed-source-ranges
+
+toc::[]
+
+You can enable and configure the `spec.endpointPublishingStrategy.loadBalancer.allowedSourceRanges` field. You can specify a list of IP address ranges for the `IngressController`. This restricts access to the Load Balancer service when the `endpointPublishingStrategy` is `LoadBalancerService`.
```

Comment:
I don't think that knowing what field you have to configure is the most important thing a user needs to know when coming to this assembly. I would remove that and leave those details to the procedures. Instead, start with what they can do - they can specify a list of IP address ranges for he Ingress Controller to restrict access to...